changelog_unreleased/javascript/10222.md

```diff
@@ -1,4 +1,4 @@
-#### Format assignments more consistently (#10222 by @thorn0)
+#### Format assignments more consistently (#10222, #10643 by @thorn0)
```

Comment:
Do we need any more updates in this changelog entry?

Reply:
What do you mean?